drivers/clock_control/clock_stm32f0_f3.c

```diff
@@ -145,6 +145,10 @@ uint32_t get_pllout_frequency(void)
 void config_enable_default_clocks(void)
 {
 #ifndef CONFIG_SOC_SERIES_STM32F3X
+
+	/* Enable PWR clock, required to access BCDR */
+	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
```

Comment:
PR says to enable the PWR clock of the stm32f3, but `#ifndef CONFIG_SOC_SERIES_STM32F3X` at line 147 wil prevent this.

Reply:
Copy/paste issue. Will fix, thanks !